packages/create-pantheon-decoupled-kit/package.json

```diff
@@ -43,8 +43,10 @@
 		"lint-staged": "lint-staged"
 	},
 	"devDependencies": {
-		"@pantheon-systems/workspace-configs": "*",
+		"@pantheon-systems/decoupled-kit-configs": "0.1.0-canary.0",
 		"@pantheon-systems/eslint-config": "*",
+		"@pantheon-systems/eslint-config-decoupled-kit": "0.1.0-canary.0",
```

Comment:
I wouldn't expect this `package.json` to change – why do we need the non-workspace configs added here?

Reply:
@CobyPear Yeah this was interesting, it is not required at all for the functionality of this feature but the monorepo linter was yelling and preventing me from committing without it there.
<img width="1430" alt="Screen Shot 2023-03-06 at 2 08 03 PM" src="https://user-images.githubusercontent.com/39682014/223247147-523342de-8d72-4cbc-a397-39be152d4c50.png">

Reply:
`lint-staged` worked for me on this branch without those dependencies, might be some config file in your tree that is looking for the published config?

Reply:
Oh yeah, if you look at the path in the error, it is looking at the config file in the template for some reason.